go.mod

```diff
@@ -255,3 +255,5 @@ require (
 	lukechampine.com/blake3 v1.2.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/livepeer/ai-worker => /home/ricks/development/livepeer/ai/ai-worker
```

Comment:
@eliteprox please remove before merge and ensure that we use the new version `v0.12.0`.